### DISCUSSION TITLE
Git compare code

### DISCUSSION THREAD

**Discussion by lotusk08** [zed-industries/zed#22233] [Feature Requests]
I really like Nova's comparison feature. Do we have any plans to do to improve this in the future?

<img width="1432" alt="Screenshot 2024-12-19 at 14 04 44" src="https://github.com/user-attachments/assets/a68cc6a4-d470-4e5f-8331-442114bfa0ff" />

**Comment by Chandler-Kluser** [comment]
[codium](https://github.com/VSCodium/vscodium) has also this feature and it works without git:

![image](https://github.com/user-attachments/assets/2f335e93-38a3-448a-931a-d9e2f54b2159)

it is a great feature!!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
